### ISSUE TITLE
Switcher on Board

### ISSUE THREAD

**Issue by DvornikovSergey** [Tradeshift/tradeshift-ui#829]
**Is your feature request related to a problem? Please describe.**
I would like to be able to add a switch to `Board`, as shown below
![image](https://user-images.githubusercontent.com/15032696/60881730-7c7b4100-a270-11e9-9abb-d55aacc1ec40.png)

**Comment by zdlm** [comment]
We reused the header in the Board. The action in the header never has a switcher. I recommend you just use a `div` and put a switcher on the right. It would be so much easier.

**Comment by DvornikovSergey** [comment]
I would like to implement it through api, but if this is a big job for you, then well, I will use the div

**Comment by zdlm** [comment]
Yes, it is a big task for us. I will keep the issue open. We will think about the feature in webcomponent version .  Thanks
